modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -182,6 +182,21 @@ public ObjectEntry getByExternalReferenceCode(String externalReferenceCode)
 			externalReferenceCode, _objectDefinition, null);
 	}
 
+	@Override
+	public Page<ObjectEntry> getByExternalReferenceCodeVersionsPage(
+			String externalReferenceCode, Pagination pagination)
+		throws Exception {
```

Comment:
Can you add the FF please?

Reply:
Or should we add the FF in `DefaultObjectEntryManagerImpl` ?

Reply:
Done!